components/jenkins/cosmosdb.tf

```diff
@@ -0,0 +1,99 @@
+locals {
+  suffix = var.env == "ptlsbox" ? "sandbox" : "prod"
+  prefix = var.env == "ptlsbox" ? "sandbox-" : ""
+}
+
+resource "azurerm_resource_group" "rg" {
+  provider = azurerm.cosmosdb
+
+  name     = "pipelinemetrics-database-${local.suffix}"
+  location = var.location
+  tags     = module.tags.common_tags
+}
+
+resource "azurerm_user_assigned_identity" "usermi" {
+  resource_group_name = var.mi_rg
+  location            = var.location
+  name                = "jenkins-${var.key_vault_name}-mi"
+  tags                = module.tags.common_tags
+}
+
+resource "azurerm_cosmosdb_account" "cosmosdb" {
+  provider = azurerm.cosmosdb
+
+  name                      = "${local.prefix}pipeline-metrics"
+  location                  = var.location
+  resource_group_name       = azurerm_resource_group.rg.name
+  offer_type                = "Standard"
+  kind                      = "GlobalDocumentDB"
+  tags                      = module.tags.common_tags
+  enable_automatic_failover = false
+  consistency_policy {
+    consistency_level = "Session"
+  }
+
+  geo_location {
+    location          = var.location
+    failover_priority = 0
+  }
+
+  geo_location {
+    location          = "UK West"
+    failover_priority = 1
+  }
+}
+
+resource "azurerm_cosmosdb_sql_database" "sqlapidb" {
+  provider = azurerm.cosmosdb
+
+  name                = var.database
+  resource_group_name = azurerm_resource_group.rg.name
+  account_name        = azurerm_cosmosdb_account.cosmosdb.name
+}
+
+resource "azurerm_cosmosdb_sql_container" "cve-reports" {
+  provider = azurerm.cosmosdb
+
+  name                  = "cve-reports"
+  resource_group_name   = azurerm_resource_group.rg.name
+  account_name          = azurerm_cosmosdb_account.cosmosdb.name
+  database_name         = azurerm_cosmosdb_sql_database.sqlapidb.name
+  partition_key_path    = "/build/git_url"
+  partition_key_version = 2
+  throughput            = 400
+
+  indexing_policy {
+    indexing_mode = "consistent"
+    included_path {
+      path = "/*"
+    }
+  }
+}
+
+resource "azurerm_cosmosdb_sql_container" "container" {
+  provider = azurerm.cosmosdb
+
+  for_each              = var.throughput
+  name                  = each.key
+  resource_group_name   = azurerm_resource_group.rg.name
+  account_name          = azurerm_cosmosdb_account.cosmosdb.name
+  database_name         = azurerm_cosmosdb_sql_database.sqlapidb.name
+  partition_key_path    = "/_partitionKey"
+  partition_key_version = 2
+  throughput            = each.value
+
```

Comment:
Is 
```indexing_policy {
  indexing_mode = "consistent"
      included_path {
        path = "/*"
      }
  }
```

Also needed for these containers? Plan is showing it will be removed